### PR TITLE
[Doc-Improvement][UpdateTeams Toolkit][2199016]

### PR DESCRIPTION
**Pull request by v-rramanna**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/create-new-project.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0d68b6b67937eb15270e24a7375b56756c695a5e/msteams-platform%2Ftoolkit%2Fcreate-new-project.md) | 100 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/83a61d28-b312-4712-a062-24c2169b6b28) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/debug-local.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0d68b6b67937eb15270e24a7375b56756c695a5e/msteams-platform%2Ftoolkit%2Fdebug-local.md) | 94 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/04e75a80-3490-4a08-b55e-b9fa3995b812) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/create-new-project.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9b23e0bfb4bea10cdbb34ac7b2e43742df23dda3/msteams-platform%2Ftoolkit%2Fcreate-new-project.md) | 92 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/491cde30-0ae0-4bdf-a64f-5d512eaaaa88) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/debug-local.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9b23e0bfb4bea10cdbb34ac7b2e43742df23dda3/msteams-platform%2Ftoolkit%2Fdebug-local.md) | 94 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/85fbc0e8-706e-48d4-8ca7-c9506dcd1ad2) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/create-new-project.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3ab8a5cf34b1bb50fa245f6c83c5b46bd5ceab48/msteams-platform%2Ftoolkit%2Fcreate-new-project.md) | 92 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fd0a5b0f-4e56-49c4-a360-d5649be697c9) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/debug-local.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3ab8a5cf34b1bb50fa245f6c83c5b46bd5ceab48/msteams-platform%2Ftoolkit%2Fdebug-local.md) | 92 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e1c40284-edd5-4064-9362-12139da13525) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/create-new-project.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2f9f5b174396771f57b339ad2928aa3546904508/msteams-platform%2Ftoolkit%2Fcreate-new-project.md) | 92 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c21a6789-997f-4f9b-99e3-c637e364d94a) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/debug-local.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2f9f5b174396771f57b339ad2928aa3546904508/msteams-platform%2Ftoolkit%2Fdebug-local.md) | 92 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1ccc1893-cf7f-4b24-a78c-4b27f01217d5) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/create-new-project.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ba3367d88946976d5dd4f0d866fbfcfde26d3e81/msteams-platform%2Ftoolkit%2Fcreate-new-project.md) | 92 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ed13a263-0552-4a9f-afe4-7c19bcd48e79) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/debug-local.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ba3367d88946976d5dd4f0d866fbfcfde26d3e81/msteams-platform%2Ftoolkit%2Fdebug-local.md) | 92 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3a7a96d5-d4ea-4527-98c1-a847569906df) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/create-new-project.md](https://github.com/MicrosoftDocs/msteams-docs/blob/90a0b118e4c0e6b90b75deb5759044043863b2b2/msteams-platform%2Ftoolkit%2Fcreate-new-project.md) | 92 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0ce0da96-2cbe-4d95-bb85-769adebf0d4c) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/debug-local.md](https://github.com/MicrosoftDocs/msteams-docs/blob/90a0b118e4c0e6b90b75deb5759044043863b2b2/msteams-platform%2Ftoolkit%2Fdebug-local.md) | 92 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c25cf46d-f9e3-4a5a-ab9d-ca2479301208) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/create-new-project.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a441308716f929fb50c1eb4231a8b1e0b8c440e2/msteams-platform%2Ftoolkit%2Fcreate-new-project.md) | 95 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/30022fce-9343-48fc-947b-675d5e92c2aa) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/debug-local.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a441308716f929fb50c1eb4231a8b1e0b8c440e2/msteams-platform%2Ftoolkit%2Fdebug-local.md) | 92 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/51f2a78d-7624-492c-aab6-70763a0dbd59) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/create-new-project.md](https://github.com/MicrosoftDocs/msteams-docs/blob/7aaf234ce94ec72a2d5848a890b88be7c14b3a91/msteams-platform%2Ftoolkit%2Fcreate-new-project.md) | 95 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/068e583c-cff4-48b7-9f87-d4316e3cec30) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/debug-local.md](https://github.com/MicrosoftDocs/msteams-docs/blob/7aaf234ce94ec72a2d5848a890b88be7c14b3a91/msteams-platform%2Ftoolkit%2Fdebug-local.md) | 92 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bcacba15-e8ce-4db8-9265-db17cd64ffa9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/create-new-project.md](https://github.com/MicrosoftDocs/msteams-docs/blob/95589d3296615e68aa557b1206ae9cd11887c886/msteams-platform%2Ftoolkit%2Fcreate-new-project.md) | 95 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2d441c70-960a-401e-93de-9c8f59b83539) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/debug-local.md](https://github.com/MicrosoftDocs/msteams-docs/blob/95589d3296615e68aa557b1206ae9cd11887c886/msteams-platform%2Ftoolkit%2Fdebug-local.md) | 93 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e2bcdafb-40fb-437e-98f6-ec191a54ecf9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/create-new-project.md](https://github.com/MicrosoftDocs/msteams-docs/blob/cee3e6e2847642c0ef2f906b25b4b53e0f5509a1/msteams-platform%2Ftoolkit%2Fcreate-new-project.md) | 95 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0c23d5cb-e2f0-4ba3-bdc6-e1d31c2a535a) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/debug-local.md](https://github.com/MicrosoftDocs/msteams-docs/blob/cee3e6e2847642c0ef2f906b25b4b53e0f5509a1/msteams-platform%2Ftoolkit%2Fdebug-local.md) | 93 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2ea173a2-97e6-4e1b-a5f5-c579bcbbdad4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/create-new-project.md](https://github.com/MicrosoftDocs/msteams-docs/blob/cee3e6e2847642c0ef2f906b25b4b53e0f5509a1/msteams-platform%2Ftoolkit%2Fcreate-new-project.md) | 95 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e13d16a5-b325-411a-b9a3-9a04a79ca991) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/debug-local.md](https://github.com/MicrosoftDocs/msteams-docs/blob/cee3e6e2847642c0ef2f906b25b4b53e0f5509a1/msteams-platform%2Ftoolkit%2Fdebug-local.md) | 93 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/eca21fe1-4a04-4ec6-a4b6-bbd6d5dd0137) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/create-new-project.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ef368df7f3070bb78c0e23391c9c270d8939ef85/msteams-platform%2Ftoolkit%2Fcreate-new-project.md) | 97 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9e04e2f9-50a6-4aeb-9ca5-78adaad620be) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/debug-local.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ef368df7f3070bb78c0e23391c9c270d8939ef85/msteams-platform%2Ftoolkit%2Fdebug-local.md) | 93 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/62e45e65-8a5f-4479-81a7-2756b04978bb) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/create-new-project.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9aec6245771aeff8e9091acfb41913ee32c82fb3/msteams-platform%2Ftoolkit%2Fcreate-new-project.md) | 97 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/acad2074-fd75-4582-b8ba-8af3b6b728de) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/debug-local.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9aec6245771aeff8e9091acfb41913ee32c82fb3/msteams-platform%2Ftoolkit%2Fdebug-local.md) | 93 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b243baec-836d-45d6-bf09-bde9efb5811b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/toolkit/create-new-project.md [hunk 1]
```diff
@@ -8,46 +8,71 @@ ms.topic: overview
...
+2. Select **DEVELOPMENT** section in Treeview and then Select **View samples**.
+   :::image type="content" source="../assets/images/teams-toolkit-v2/sample1.png" alt-text="View samples":::
+1. The **Samples** gallery displays.
 
-    :::image type="content" source="../assets/images/teams-toolkit-v2/view-samples.png" alt-text="View samples":::
+   :::image type="content" source="../assets/images/teams-toolkit-v2/samples2.png" alt-text="View samples2":::
+   
 
-You can explore and download samples and either run your app locally or remotely to preview in Teams web client. Follow the instructions for each sample, or select **View on GitHub** to open the sample within the `TeamsFx Samples repository` and browse the source code.
+Explore samples to quickly get started with concepts and code examples. Follow the instructions for each sample, or select **View on GitHub** to open the sample in the `TeamsFx Samples` repository and browse the source code.
```
Comment (v-abirade) [inline msteams-platform/toolkit/create-new-project.md:60]: You can remove the world quickly. Please update information on the graphics and documents available to guide developers to create app.

File: msteams-platform/toolkit/create-new-project.md [hunk 2]
```diff
@@ -8,46 +8,71 @@ ms.topic: overview
...
+### View samples
 
-You can create a new app by exploring **View samples** and selecting an existing sample. The selected sample may already have some functionality, for example a to-do list with an Azure backend, or an integration with the Microsoft Graph Toolkit.
+You can create a new Teams app by selecting an existing sample. Selecting an existing sample provides you with an inbuilt functionality. For example, a **Todo List with backend on Azure**, or **Hello World Bot with SSO**.<br>
 
- 1. Open **Teams Toolkit** from Microsoft Visual Studio Code.
- 1. Select **DEVELOPMENT** section in Treeview.
- 1. Select **View samples**. The sample gallery appears as shown in the following image:
+1. Select **Teams Toolkit** :::image type="icon" source="../assets/images/teams-toolkit-v2/teams-toolkit-sidebar-icon.PNG" border="true"::: icon in the Visual Studio Code sidebar.
+2. Select **DEVELOPMENT** section in Treeview and then Select **View samples**.
```
Comment (v-abirade) [inline msteams-platform/toolkit/create-new-project.md:53]: Please confirm, if this step is correct:
2. Select **DEVELOPMENT** section in Treeview and then Select **View samples**.
Can we select DEVELOPMENT? or do we need to select **View Samples** listed under **DEVELOPMENT** section.

**Review by v-abirade — CHANGES_REQUESTED** [review]
@v-rramanna, please work on the comments.

File: msteams-platform/toolkit/debug-local.md [hunk 3]
```diff
@@ -7,9 +7,9 @@ ms.topic: overview
 ms.date: 03/21/2022
 ---
 
-# Debug your Teams app locally
+# Debug Teams app locally
 
-Teams Toolkit helps you to debug and preview your Teams app locally. Debug is the process of checking, detecting, and correcting issues or bugs to ensure the program runs successfully. Visual Studio Code allows you to debug tab, bot, message extension, and Azure Functions. Teams Toolkit supports the following debug features:
+ Debug is the process of resolving the conflicts to ensure the program runs successfully. Visual Studio Code allows you to debug tab, bot, message extension, and Azure Functions. Teams Toolkit helps you to debug and preview your Teams app locally and also supports the following debug features:
```
Comment (v-abirade) [inline msteams-platform/toolkit/debug-local.md:12]: Please retain: Teams Toolkit helps you to debug and preview your Teams app locally. Debug is the process of checking, detecting, and correcting issues or bugs.

File: msteams-platform/toolkit/debug-local.md [hunk 4]
```diff
@@ -18,25 +18,25 @@ Teams Toolkit helps you to debug and preview your Teams app locally. Debug is th
 * [Stop debugging](#stop-debugging)  
 
 
-During the debug process, Teams Toolkit automatically starts app services, launches debuggers, and sideloads the Teams app. The Teams app is available for preview in Teams web client locally after debugging. You can also customize debug settings to use your bot endpoints, development certificate, or debug partial component to load your configured app.
+During the debug process, Teams Toolkit automatically starts app services, launches debuggers, and sideloads the Teams app. You can also customize the debug settings to use your bot endpoints, development certificate, or debug partial component to load your configured app. The Teams app is available for preview in Teams web client locally after the debugging.
 
 ## Prerequisite
```
Comment (v-abirade) [inline msteams-platform/toolkit/debug-local.md:23]: Do we need to update prerequisite in all the files?

File: msteams-platform/toolkit/debug-local.md [hunk 5]
```diff
@@ -18,25 +18,25 @@ Teams Toolkit helps you to debug and preview your Teams app locally. Debug is th
...
 
 Install the [latest version of Teams Toolkit](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension).
 
-## Key features of Teams Toolkit
-
+## Key debug features
+The following list shows the primary debug features used in Teams Toolkit.
```
Comment (v-abirade) [inline msteams-platform/toolkit/debug-local.md:28]: There is no need to add the sentence as the heading provides the required information.

File: msteams-platform/toolkit/debug-local.md [hunk 6]
```diff
@@ -18,25 +18,25 @@ Teams Toolkit helps you to debug and preview your Teams app locally. Debug is th
...
 #### Start debugging
 
-You can perform single operation, select **F5** to start debugging. The Teams Toolkit starts checking prerequisites, registering Azure Active Directory app, registering Teams app, registering bot, starting services, and launching browser.
+You can select **F5** to start debugging. The Teams Toolkit starts checking prerequisites, registering Azure Active Directory app, registering Teams app, registering bot, starting services, and launching browser.
 
 #### Multi-target debugging
 
-Teams Toolkit utilizes multi-target debugging feature to debug tab, bot, message extension, and Azure Functions at the same time.
+You can select debut option under **DEVELOPMENT** activity bar, Teams Toolkit utilizes multi-target debugging feature to debug tab, bot, message extension, and Azure Functions at the same time.
```
Comment (v-abirade) [inline msteams-platform/toolkit/debug-local.md:35]: Check for Typos.

File: msteams-platform/toolkit/debug-local.md [hunk 7]
```diff
@@ -54,16 +54,17 @@ When you complete local debug, you can select **Stop** or **Disconnect** from th
 
 ## Debug your Teams app locally
 
+
 #### 1. Set up your Teams Toolkit
 
-Complete the following steps to debug your app after creating a new app using the Teams Toolkit:
+Follow the steps to debug your app once you create a new app:
```
Comment (v-abirade) [inline msteams-platform/toolkit/debug-local.md:60]: Please correct the sentence.

File: msteams-platform/toolkit/debug-local.md [hunk 8]
```diff
@@ -156,15 +157,15 @@ Launches the debug configurations defined in `.vscode/launch.json` as follows:
 
 :::image type="content" source="../assets/images/teams-toolkit-v2/debug/launch-debuggers.png" alt-text="Launch debugger":::
 
-The following table lists the debug configuration names and types for project with tab app and bot app:
+The debug configuration names and types for project with Tab app and Bot app:
```
Comment (v-abirade) [inline msteams-platform/toolkit/debug-local.md:160]: Please rephrase to the previous sentence:
The following table lists the debug configuration names and types for project with tab app and bot app:

File: msteams-platform/toolkit/debug-local.md [hunk 9]
```diff
@@ -156,15 +157,15 @@ Launches the debug configurations defined in `.vscode/launch.json` as follows:
...
 
 |  Component |  Debug configuration name  | Debug configuration type |
 | --- | --- | --- |
 |  Tab |  **Attach to Frontend (Edge)** or  **Attach to Frontend (Chrome)**  |  pwa-msedge or pwa-chrome  |
 |  Bot or message extensions |   **Attach to Bot** |  pwa-node |
 | Azure Functions |   **Attach to Backend** |  pwa-node |
 
-The following table lists the debug configuration names and types for project with bot app and without tab app:
+The debug configuration names and types for project with Bot app:
```
Comment (v-abirade) [inline msteams-platform/toolkit/debug-local.md:168]: Please retain.

File: msteams-platform/toolkit/debug-local.md [hunk 10]
```diff
@@ -174,15 +175,16 @@ The following table lists the debug configuration names and types for project wi
 
 #### Sideloads the Teams app
 
-The configuration **Attach to Frontend** or **Launch Bot** launches a new Edge or Chrome browser instance and opens a web page to load Teams client. After the Teams client is loaded, Teams sideloads the Teams app controlled by the sideloading url defined in the launch configurations [Microsoft Teams](https://teams.microsoft.com/l/app/>${localTeamsAppId}?installAppPackage=true&webjoin=true&${account-hint}).  When Teams client loads in the web browser, select **Add** or select one from the dropdown list as per your requirement.
+The configuration **Attach to Frontend** or **Launch Bot** launches a new browser instance and opens a web page to load the Teams client. After the Teams client is loaded, Teams sideloads the Teams app controlled by the sideloading url defined in the launch configurations [Microsoft Teams](https://teams.microsoft.com/l/app/>${localTeamsAppId}?installAppPackage=true&webjoin=true&${account-hint}).  When Teams client loads in the web browser, select **Add** or select any one of the dropdown list option.
```
Comment (v-abirade) [inline msteams-platform/toolkit/debug-local.md:178]: When Teams client loads in the web browser, select **Add** or select the required dropdown list option.

**Review by v-abirade — CHANGES_REQUESTED** [review]
Please make changes

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/create-new-project.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8a0785f117e9465b1d6feca6e30fd97bfadb679c/msteams-platform%2Ftoolkit%2Fcreate-new-project.md) | 100 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/630e0166-3448-47c4-a01e-7ba341e87e48) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/debug-local.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8a0785f117e9465b1d6feca6e30fd97bfadb679c/msteams-platform%2Ftoolkit%2Fdebug-local.md) | 93 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/37ac2ec9-e375-483c-860d-446c4ec9b780) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by surbhigupta12** [comment]
Closing this PR as it is obsolete.

**Pull request closed without merging**